WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -111,7 +109,12 @@ private static uint256 CalculateHash(Coin coin, BitcoinSecret bitcoinSecret, uin
 			var ownershipProof = OwnershipProof.GenerateCoinJoinInputProof(
 				bitcoinSecret.PrivateKey,
 				new CoinJoinInputCommitmentData("CoinJoinCoordinatorIdentifier", roundId));
-			return new Alice(coin, ownershipProof).Id;
+
+			return StrobeHasher.Create(ProtocolConstants.AliceStrobeDomain)
```

Comment:
How about having a static method in `WalletWasabi.WabiSabi.Helpers.AliceHelpers.CalculateHash()` to avoid code dupl?

Reply:
If we are going to have an static method then I would prefer to have it in `Alice`, something like `public static uint256 CalculateHash(Coin, OwnershipProof)` and then we can use it from here and also from the `Alice` class itself.